hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/config/TokenServiceConfig.java

```diff
@@ -23,6 +23,9 @@
  * Provides configuration for the token service. Currently, it just has the maximum number of custom fees allowed.
  * In the future all the configurations needed by handlers in the token service will be added here.
  * @param maxCustomFeesAllowed the maximum number of custom fees allowed
+ * @param maxTokenRelsPerInfoQuery the maximum number of token relation per info query
  */
 @ConfigData("token")
-public record TokenServiceConfig(@ConfigProperty(defaultValue = "10") long maxCustomFeesAllowed) {}
+public record TokenServiceConfig(
```

Comment:
This config will be deleted. Please use `TokensConfig`

Reply:
Replaced all code with TokensConfig.